app/models/place.rb

```diff
@@ -27,5 +30,15 @@ def update_address(place, raw_place)
   def set_slug
     self.slug = name.parameterize
   end
-   
+
+  def self.slug_for_show(place_id)
+    @client = GooglePlaces::Client.new(ENV['GOOGLE_KEY'])
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.